models/user.php

```diff
@@ -21,4 +30,33 @@ public function register(){
 		}
 		return;
 	}
+	public function login(){
+		// Sanitize POST
+		$post = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+
+		$password = md5($post['password']);
+
+		if($post['submit']){
+			// Compare Login
+			$this->query('SELECT * FROM users WHERE email = :email AND password =:password');
+			$this->bind(':email', $post['Email']);
+			$this->bind(':password', $password);
+			
+			$row = $this->single();
+
+			if($row){
```

Comment:
No need for if else, first check if there is a row, if not return the message
```suggestion
			if(!$row){
			    Messages::setMsg('Incorrect Login', 'error');
			    return;
			}
```